### ISSUE TITLE
Apollo 9 LEM RCS and Engine dont activate

### ISSUE THREAD

**Issue by RL77** [dseagrav/NASSP#404]
I apologize for writing here again, but I do not know if this problem is caused by something I did wrong, or by a bug. Also your support here is more fast than in orbiter forum  . I was able to finish the checklist in time for the first burn of the LEM in Apollo 9, but near the ignition the RCS do not activate, and the LEM engine does not fire when the time comes. Can someone tell me what might be wrong?
I would also like to know information about certain yellow messages in the CSM/LEM checklist, such as "Install tunnel hardware", "connect hoses to AGP". How do I know which of these messages are just to inform you that this task takes place, or to perform the task. Is there anything you can read to know this?
Regarding Project Apollo MFD I have now started to use, since apollo 9 requires it. I have already found instructions on this page:

http://nassp.sourceforge.net/wiki/ProjectApolloMFD

But it does not answer all my questions in how to work with this, regarding what we need to do on Apollo 9. The checklist says what we have to do in this MFD when it is needed, but most of the time I do what the checklist says, but notting realy does happen the MFD display. Also I can see than we can transfer the astronauts in the MFD. Do I need to transfer 2 astronauts to start the LEM checklist?
I usually do not  ask so many questions, because I always go looking for instrutions to read about, but I have some difficulty in finding the right documents to read on Apollo 9 mission. I'm using the Apollo 9 flight plane, but I do not think it will be enough for everything.

https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/19710011038.pdf

Can you give me some help in finding more instructions for Apollo 9?

Thank you,
Best regards

**Comment by danmaas** [comment]
Hi RL - Did you hit PRO on the DSKY during the last 5 seconds before ignition? ("AUTO IGN" in the checklist). I think a single press is sufficient, although I usually hold the key until TIG.

For the MFD question, are you using the "MCC" version of this scenario, where a simulated ground control staff sends you maneuvers and state vectors via uplinks? Or are you using the "non MCC" version where you have to compute maneuvers yourself?

Also, I had a similar problem with the CSM RCS, and fixed it using the advice here:

https://www.orbiter-forum.com/showthread.php?p=576751
https://www.orbiter-forum.com/showthread.php?t=39903

Basically, double-check the RCS-related circuit breakers are all closed, and then try these switches at the bottom-left corner of panel 2:
RCS CMD - ON
RCS TRNFR - SM

**Comment by RL77** [comment]
hello

I'm doing the mission with MCC. I only have problems with the LEM engine and RCS. CSM works great! This is basically the first activation of the LEM in apollo9, and I am referring to the first burn that it makes attached to the CSM. Yes I hit pro and verified if it had effect, it is the same as the procedure in the CSM.

Thank you

**Comment by jalexb88** [comment]
Hey RL77,

About your 1st issue, be sure these 2 switches are set correctly:

GUID CONT - PGNS
MODE CONTROL PGNS - AUTO

That would cause your issue if those were not set. If it is not that then I would need to take a look at your .scn file if you could post it on here.

About the yellow messages, a lot of them such as "Install tunnel hardware" are referring to the probe and drogue which in reality was physically removed from the tunnel and stowed after docking. In NASSP however this is not yet simulated. All that works is simply the opening and closing of the hatch itself.

What exactly do you mean by "notting realy does happen the MFD display"? When you add crew members with the ECS page "CRW" button, does it not show up on the page itself?

**Comment by jalexb88** [comment]
Was the PIPA Bias check part the checklist MFD flight plan for Apollo 9? I am not the most qualified to answer this one, but I believe that the simulated accelerometers in the NASSP IMU are essentially "perfect" so that means we don't need to do the PIPA procedure in our case. Doing so may even be detrimental to accuracy. Niklas (indy91) may be better to answer that one though.

I believe the CSM right now only simulates the crew in suit right now, so that is why there is no option for astronauts on cabin for the CSM at this time.

**Comment by RL77** [comment]
Hello,

Yes, PIPA Bias is part of the checklist on apollo 9, but it appears as a yellow message in NASSP checklist. I did not know if it was to do or not, but as it was a procedure in the AGC, I thought it had to be done. So I'm going to go back and start activating RCS without doing PIPA Bias Check, since you say it may even detrimental to accuracy.
 I'm still newby and only  have the missions of apollo  7 and 8  done. It's my 1st mission with LEM.

Thanks for your help :)

**Comment by jalexb88** [comment]
No problem! ;) 

Yeah, I have literally never done a PIPA bias check in the many years I've used NASSP so I am not really sure what it would do to be honest, but I do know its not required. I guess if you really want to for the sake of doing the procedure, you can do it and load a bias of 0 and that should ensure you dont lose accuracy.

**Comment by RL77** [comment]
Just to say,  that I was able to do the 1st burn with LEM in apollo 9 
Thanks for the support.

**Comment by jalexb88** [comment]
Glad to hear! You can press "close issue" on the bottom right if you want at this time as it is now addressed. Feel free to ask more questions on the forums if possible ;) Of course if you do run into a bug/issue then here is the place for that.

**Comment by RL77** [comment]
Ok thanks!
